<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCSettingAutoReadInputChannelConfigDlg</class>
 <widget class="QDialog" name="BCSettingAutoReadInputChannelConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>电脑数量</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="m_pPCCountLineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>电脑起始ID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="m_pPCBeginIDLineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>IPVideo数量</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="m_pIPVedioCountLineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>IPVideo起始ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="m_pIPVedioBeginIDLineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>Video数量</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="m_pVedioCountLineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>Video起始ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="m_pVedioBeginIDLineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>高清数量</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="m_pHDCountLineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>高清起始ID</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="m_pHDBeginIDLineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pOKBtn">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pCancelBtn">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QCheckBox" name="m_pIsUseCheckBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>自动检测输入信息</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pIsUseCheckBox</tabstop>
  <tabstop>m_pPCCountLineEdit</tabstop>
  <tabstop>m_pPCBeginIDLineEdit</tabstop>
  <tabstop>m_pIPVedioCountLineEdit</tabstop>
  <tabstop>m_pIPVedioBeginIDLineEdit</tabstop>
  <tabstop>m_pVedioCountLineEdit</tabstop>
  <tabstop>m_pVedioBeginIDLineEdit</tabstop>
  <tabstop>m_pHDCountLineEdit</tabstop>
  <tabstop>m_pHDBeginIDLineEdit</tabstop>
  <tabstop>m_pOKBtn</tabstop>
  <tabstop>m_pCancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
